internal/api/dto/invoice.go

```diff
@@ -335,6 +339,8 @@ type InvoiceResponse struct {
 	PaymentStatus   types.PaymentStatus        `json:"payment_status"`
 	Currency        string                     `json:"currency"`
```

Comment:
Need comments here as well on the fields (and maybe copy the same on invoice model) for them to be available in docs